debian/changelog

```diff
@@ -1,3 +1,16 @@
+daos (2.5.100-10) unstable; urgency=medium
+  [ Tomasz Gromadzki ]
+  * Update PMDK to 2.0.0
+    * Remove libpmemblk from dependencies.
+    * Start using BUILD_EXAMPLES=n and BUILD_BENCHMARKS=n instead of patches.
+    * Stop using BUILD_RPMEMM=n and NDCTL_DISABLE=y.
```

Comment:
Two things:

- `BUILD_RPMEMM` too many Ms
- I believe it will be good to write down your reasons.

```suggestion
    * Stop using BUILD_RPMEM=n (removed) and NDCTL_DISABLE=y (invalid).
```